ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/utils.ts

```diff
@@ -11,3 +11,17 @@ export const cvesDiscoveredSinceLabelMap: Record<CVESDiscoveredSince, string> =
     SINCE_LAST_REPORT: 'Last successful scheduled run report',
     START_DATE: 'Custom start date',
 };
+
+export const commaSeparateWithAnd = (arr: string[]) => {
+    if (arr.length === 0) {
+        return '';
+    }
+    if (arr.length === 1) {
+        return arr[0];
+    }
+    const last = arr.pop();
+    if (!last) {
```

Comment:
`arr.pop()` apparently returns a type of `string | undefined` so I had to do this because of that